### DISCUSSION TITLE
Support HTML as checkbox text

### DISCUSSION THREAD

**Discussion by letmaik** [vueform/vueform#215] [Ideas]
Checkboxes are often used to accept something like terms and conditions. For that, it would be useful to hyperlink some words. The work-around is to use two columns with sizes 1 and 11 where the left one is the checkbox with empty text, and on the right there's a static element with HTML. This works but the spacing is too wide, and it also complicates the code/layout.

It would be great to have an additional `html` option in CheckboxElement that takes precedence over `text`. The main difference in behaviour is that the text wouldn't be clickable anymore (to tick the checkbox) so the user would have to specifically click the checkbox, but I think that's ok.

**Comment by adamberecz** [comment]
Right now this is possible in Vueform, but it's true that only the `simple` preset shows a rich text editor for checkbox text in the builder. If this is meant to be for the builder I'll transfer the discussion there.

**Reply by letmaik** [reply]
I see, I wasn't aware that `text` also accepts HTML. I think the docs at https://vueform.com/reference/checkbox-element#option-text should mention it.

The builder itself swallows the HTML tags in the source view:

![image](https://github.com/vueform/vueform/assets/530988/0a085628-b0e7-4d72-adc9-24e9249356d1)

**Reply by adamberecz** [reply]
Thanks for noting - added to the docs. Nice catch for the other issue - fixed.
